lnrpc/rpc.proto

```diff
@@ -2371,6 +2371,12 @@ message Invoice {
 
     /// List of HTLCs paying to this invoice [EXPERIMENTAL].
     repeated InvoiceHTLC htlcs = 22 [json_name = "htlcs"];
+
+    /**
+    Indicates if this invoice was a spontaneous payment that arrived via keysend
+    [EXPERIMENTAL].
+    */
+    bool key_send = 24 [json_name = "key_send"];
```

Comment:
naming suggestion: `is_keysend` to indicate boolean, but not a blocker

Reply:
I was thinking about the go convention of leaving off `is`. But changed.